generators/cypress/templates/src/test/javascript/cypress/support/index.ts.ejs

```diff
@@ -46,3 +46,10 @@ Cypress.Cookies.defaults({
 preserve: ['XSRF-TOKEN','SESSION', 'JSESSIONID']
 })
 <%_ } _%>
+
+// https://docs.cypress.io/api/events/catalog-of-events#Uncaught-Exceptions
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
Don’t add to angular see https://github.com/jhipster/generator-jhipster/pull/14957/commits/be480fed73dc509179368ea7c86786eb0ed69b1d

Reply:
ok

Reply:
`clientFrameworkVue` is missing. The above commit adds it too.

Reply:
oups